README.md

```diff
@@ -3,228 +3,29 @@
-enable-debug-api: true
-data-dir: /tmp/bee/node2
-password: some pass phze
-verbosity: trace
-tracing: true
-```
-
-### Starting the first node
-
-The first node address will be used for other nodes to discover themselves. It is usually referred as the `bootnode address`.
-
-```sh
-bee --config node1.yaml start
-```
-
-When the node starts it will print out some of its `p2p addresses` in a multiaddress form like this: `/ip4/127.0.0.1/tcp/7071/p2p/16Uiu2HAm2LXfYsY9pXtgGdQ8oPb3bAkxwpfBE6AMzcscH1UkQLZM`. This address is the `bootnode address`.
-
-### Starting other nodes
-
-Other nodes should be started by providing the bootnode address to them, so that they can connect to each other:
-
-```sh
-bee --config node2.yaml start --bootnode /ip4/127.0.0.1/tcp/7071/p2p/16Uiu2HAm2LXfYsY9pXtgGdQ8oPb3bAkxwpfBE6AMzcscH1UkQLZM
-bee --config node3.yaml start --bootnode /ip4/127.0.0.1/tcp/7071/p2p/16Uiu2HAm2LXfYsY9pXtgGdQ8oPb3bAkxwpfBE6AMzcscH1UkQLZM
-bee --config node4.yaml start --bootnode /ip4/127.0.0.1/tcp/7071/p2p/16Uiu2HAm2LXfYsY9pXtgGdQ8oPb3bAkxwpfBE6AMzcscH1UkQLZM
-...
-```
-
-## Getting node addresses
-
-Every node can provide its overlay address and underlay addresses by reading the logs when the node starts or through Debug API. For example, for the node 1 started with configuration above:
-
-```sh
-curl localhost:6061/addresses
-```
-
-Debug API is not started by default and has to be explicitly enabled with `--enable-debug-api --debug-api-addr 127.0.0.1:6061` command line flags or options.
-
-It will return a response with addresses:
-
-```json
-{
-  "overlay": "7ecf777fdab7553fbc4db7f265a7bd80d27b171babe931bc61e9d7966974ef47",
-  "underlay": [
-    "/ip6/::1/udp/7071/quic/p2p/16Uiu2HAm2LXfYsY9pXtgGdQ8oPb3bAkxwpfBE6AMzcscH1UkQLZM",
-    "/ip4/127.0.0.1/tcp/7071/p2p/16Uiu2HAm2LXfYsY9pXtgGdQ8oPb3bAkxwpfBE6AMzcscH1UkQLZM",
-    "/ip4/127.0.0.1/udp/7071/quic/p2p/16Uiu2HAm2LXfYsY9pXtgGdQ8oPb3bAkxwpfBE6AMzcscH1UkQLZM",
-    "/ip6/::1/tcp/7071/p2p/16Uiu2HAm2LXfYsY9pXtgGdQ8oPb3bAkxwpfBE6AMzcscH1UkQLZM",
-  ]
-}
-```
-
-## Testing a connection with PingPong protocol
-
-To check if two nodes are connected and to see the round trip time for message exchange between them, get the overlay address from one node, for example local node 2:
-
-```sh
-curl localhost:6062/addresses
-```
-
-Make sure that Debug API is enabled and address configured as in examples above.
-
-And use that address in the Debug API call on another node, for example, local node 1:
-
-```sh
-curl -XPOST localhost:6061/pingpong/d4440baf2d79e481c3c6fd93a2014d2e6fe0386418829439f26d13a8253d04f1
-```
-
-## Generating protobuf
-
-To process protocol buffer files and generate the Go code from it two tools are needed:
-
-- protoc - https://github.com/protocolbuffers/protobuf/releases
-- protoc-gen-gogofaster - https://github.com/gogo/protobuf
-
-Makefile rule `protobuf` can be used to automate `protoc-gen-gogofaster` installation and code generation:
-
-```sh
-make protobuf
-```
```

Comment:
We can remove this now, but should find a better place as it contains an important information for development.

Reply:
It is in the docs ;)

Reply:
Yes, but it would be better to be closer to the code, in this repository as it is related only to the code, not to the usage of swarm.

Reply:
I understand, I added it back for now. It's not a big deal to have some things at two places. Any more things which you think should be added back?

Reply:
Actually, I think that it can be removed now, as it will take a lot of space in the readme. We can put it later somewhere else. Sorry, I was just commenting on the context of this section. It is better to remove it from readme as readme is shorter now.

Reply:
Removed it again :)

Reply:
>  We can put it later somewhere else

Any ideas on where to put it?

Reply:
We can have a separate file for it. But we can make that later.